### PR TITLE
Design guideline update - Hand menu 

### PR DESCRIPTION
**Pull request by cre8ivepark**
Updated Hand Menu design guideline, based on our learnings and updates added to MRTK v2.4.0

### CODE REVIEW

**Comment by cre8ivepark** [comment]
Ken is doing some additional updates.

**Comment by jessemcculloch** [comment]
@cre8ivepark - Should @hferrone hold off on reviewing then?

**Comment by cre8ivepark** [comment]
@jessemcculloch @hferrone Update from Ken merged and ready to review!

File: mixed-reality-docs/hand-menu.md [hunk 1]
```diff
@@ -11,38 +11,73 @@ keywords: hand, menu, button, quick access, layout
 
 ![Ulnar side hand location](images/UX/UX_Hero_HandMenu.jpg)
 
-Hand menus allow users to quickly bring up hand-attached UI for frequently used functions. 
+The hand menu is one of the most unique UX patterns in HoloLens 2. It allows you to quickly bring up hand-attached UI. Since it is accessible anytime and can be shown and hidden easily, it is great for quick actions.
```
Comment (hferrone) [inline mixed-reality-docs/hand-menu.md:14]: Change "it is" to "it's"

File: mixed-reality-docs/hand-menu.md [hunk 2]
```diff
@@ -11,38 +11,73 @@ keywords: hand, menu, button, quick access, layout
...
 
-Below are the best practices we have found for hand menus. You can also find an example scene demonstrating the hand menu in [MRTK](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Documentation/README_Solver.md#hand-menu-with-handconstraint-and-handconstraintpalmup).
+> [!VIDEO https://www.microsoft.com/en-us/videoplayer/embed/RE4AJAg]
+
+Below are the best practices we have found for hand menus. You can also find an example scene demonstrating the hand menu in [MRTK](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_HandMenu.html).
```
Comment (hferrone) [inline mixed-reality-docs/hand-menu.md:18]: You'll find our recommended best practices for working with hand menus in the list below.

File: mixed-reality-docs/hand-menu.md [hunk 3]
```diff
@@ -11,38 +11,73 @@ keywords: hand, menu, button, quick access, layout
...
+Do not assume both of the user's hands are always available. Consider a wide range of contexts when one or both hands are not available, and make sure your design accounts for those situations. To support a one-handed hand menu, you can try transitioning the menu placement from hand-locked to world-locked when the hand flips (goes palm down). For hands-free scenarios, consider using a voice command to invoke the hand menu.
 
-**F. Avoid adding buttons near the wrist (system home button):**
-If the hand menu buttons are placed too close to the home button, it may get accidentally triggered while interacting with the hand menu.
+**Avoid adding buttons near the wrist (system home button)**
 
-**G. Test, test, test:**
-People have different bodies, with different thresholds for comfort and discomfort, etc. Be sure to test out your design on and get feedback from a variety of people.
+If the hand menu buttons are placed too close to the home button, it may get accidentally triggered while interacting with the hand menu.
```
Comment (hferrone) [inline mixed-reality-docs/hand-menu.md:44]: "it may accidentally trigger while..."

File: mixed-reality-docs/hand-menu.md [hunk 4]
```diff
@@ -11,38 +11,73 @@ keywords: hand, menu, button, quick access, layout
...
 
 <br>
 
+## Hand menu with large and complex UI controls
+<img src="images/UX/HandMenu_SizeExample.png" alt="HoloLens perspective of a menu system that always faces the user" width="940px">
+It is recommended to limit the number of buttons or UI controls on hand-attached menus. This is because extended interaction with a large number of UI elements can cause arm fatigue. If your experience requires a large menu, provide an easy way for the user to world lock the menu. One technique we recommend is to world-lock then menu when the hand drops or flips away from the user. A second technique is to allow the user to directly grab the menu with the other hand. When the user releases the menu, the menu should world lock. This way, a user can interact with various UI elements comfortably and confidently over an extended period of time.
```
Comment (hferrone) [inline mixed-reality-docs/hand-menu.md:50]: It's

File: mixed-reality-docs/hand-menu.md [hunk 5]
```diff
@@ -11,38 +11,73 @@ keywords: hand, menu, button, quick access, layout
...
+It is recommended to limit the number of buttons or UI controls on hand-attached menus. This is because extended interaction with a large number of UI elements can cause arm fatigue. If your experience requires a large menu, provide an easy way for the user to world lock the menu. One technique we recommend is to world-lock then menu when the hand drops or flips away from the user. A second technique is to allow the user to directly grab the menu with the other hand. When the user releases the menu, the menu should world lock. This way, a user can interact with various UI elements comfortably and confidently over an extended period of time. 
+
+When the menu is world-locked, make sure to provide a way to move the menu, and close the menu when it is no longer needed. Make the menu movable by providing handles on the sides or top of menu. Add a close button to allow the menu to close. Allow for the menu to re-attach to the hand when the user hand faces the user. We also recommend requiring that the users gazes at their hand to prevent false activations (see below).
```
Comment (hferrone) [inline mixed-reality-docs/hand-menu.md:52]: it's

**Comment by cre8ivepark** [comment]
@hferrone  Applied the feedback. Thank you!

**Comment by hferrone** [comment]
@jessemcculloch This one's good to go if you are :)

**Pull request merged**